tests/end-to-end/backup-restore-test/deploy/chart/backup-test/values.yaml

```diff
@@ -0,0 +1,5 @@
+containerRegistry:
+  path: docker.io/k15r
```

Comment:
why do you use your docker. I guess we can put gcr here.

Reply:
i agree, that was just for testing until i have an image created by our system